docs/dev/reference/apis/sessions.md

```diff
@@ -60,6 +60,118 @@ If a resource was used by client A and then by client B, and client A disconnect
 
 A disconnected client will attempt to establish a new session immediately prior to the next operation it performs.
 
+#### Change the session timeout
+
+The default session length is 20 seconds.
+To change the session timeout length, pass the `timeout` parameter to the `DialOptions` object:
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+```python {class="line-numbers linkable-line-numbers" data-line="1"}
+opts = RobotClient.Options(dial_options=DialOptions(timeout=10)).with_api_key(
+  # Replace "<API-KEY>" (including brackets) with your machine's api key
+  api_key='<API-KEY>',
+  # Replace "<API-KEY-ID>" (including brackets) with your machine's api key id
```

Comment:
```suggestion
  # Replace "<API-KEY>" (including brackets) with your machine's API key
  api_key='<API-KEY>',
  # Replace "<API-KEY-ID>" (including brackets) with your machine's API key ID
```
[nit/opt] For all of these, correct caps would be nice but I know app doesn't generate it that way

Reply:
fixed in app as well.